code/datums/materials/basemats.dm

```diff
@@ -72,7 +72,7 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	strength_modifier = 1.2
 	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE, MAT_CATEGORY_BASE_RECIPES = TRUE, MAT_CATEGORY_ITEM_MATERIAL=TRUE)
 	sheet_type = /obj/item/stack/sheet/mineral/gold
-	value_per_unit = 1.25
+	value_per_unit = 125 / SHEET_MATERIAL_AMOUNT
```

Comment:
Just went and checked against the old numbers vs new numbers, you missed the value on gold a bit.
![image](https://github.com/tgstation/tgstation/assets/41715314/12a849ee-c140-489d-a020-66d4147833eb)
Gold is supposed to be the same value as titanium, not uranium.
Swap the two and you should be good

Reply:
But titanium is 125 / sheet_material_amount
The same as gold
I don't get it

Reply:
Yeah I'm confused as well both gold and titanium have
125/SHEET_MATERIAL_AMOUNT so they already have the same value and uranium has
100/SHEET_MATERIAL_AMOUNT

Reply:
Nah you right, I'm blind